packages/boxel/addon/components/boxel/tab-bar/index.css

```diff
@@ -42,7 +74,7 @@
   }
 }
 
-.boxel-tab-bar__item > a:hover {
+.boxel-tab-bar__item > a:hover .boxel-tab-bar__item-text {
```

Comment:
This rule will not be applied because the `a` tag is the `.boxel-tab-bar__item`. We can safely remove it and preserve existing behaviour.

Reply:
it’s true, this is a relic, thanks!